app/views/content/train-to-be-a-teacher/teacher-degree-apprenticeships.md

```diff
@@ -47,20 +47,24 @@ The providers offering teacher degree apprenticeships are:
 * Xavier Teach Southeast, in partnership with the University of Sussex
 * University of Derby
 
-To view all courses for the TDA in secondary maths, please visit [Find Teacher Training](https://www.gov.uk/find-teacher-training-courses).
+[View all courses for the TDA in secondary maths](https://www.gov.uk/find-teacher-training-courses).
 
 ## Who’s eligible to apply for a teacher degree apprenticeship?
 Teacher degree apprenticeships will be open to anyone who would like to gain a degree and become a qualified teacher. This includes, but is not limited to, teaching assistants, school leavers or people looking to change careers.
 
-To be eligible to apply for a teacher degree apprenticeship you’ll need the following qualifications:
+You can apply for a place on a teacher degree apprenticeship if you meet specific eligibility requirements.
```

Comment:
Does this need to say `specific`?

Or could it just say '... if you are eligible. You must be:` and then get rid of line 57 - seems a bit duplicate?